moveos/moveos-stdlib/moveos-stdlib/sources/context.move

```diff
@@ -105,21 +106,10 @@ module moveos_std::context {
     }
 
     #[private_generics(T)]
-    /// Remove object from object store
-    public fun remove_object<T: key>(self: &mut Context, object_id: ObjectID): Object<T> {
-        storage_context::remove<T>(&mut self.storage_context, object_id)
-    }
-
-    #[private_generics(T)]
-    public fun remove_object_with_ref<T: key>(self: &mut Context, object_ref: ObjectRef<T>): Object<T> {
-        let object_id = object_ref::into_id(object_ref);
-        storage_context::remove<T>(&mut self.storage_context, object_id)
-    }
-
-    #[private_generics(T)]
-    /// Add object to object store
-    public fun add_object<T: key>(self: &mut Context, obj: Object<T>) {
-        storage_context::add<T>(&mut self.storage_context, obj)
+    /// Remove object from object store, and unpack the Object
+    public fun remove_object<T: key>(self: &mut Context, object_id: ObjectID): (ObjectID, address, T) {
```

Comment:
After remove object, the original ObjectRef reference still exists. If call `object_ref::borrrow` or `object_ref::borrrow_mut`, the underlying raw_table will throw a NotFound error code. The Rust life cycle management mechanism fails in this dimension. 

 Should we consider adding track ObjectRef?

Reply:
Record a TODO about this

https://github.com/rooch-network/rooch/blob/e79f988fbac899e7c60fdc745fac091fd9210a9f/moveos/moveos-stdlib/moveos-stdlib/sources/object_ref.move#L22-L25

Currently, before using ObjectRef, the dev needs to call `object_ref::exist_object` to ensure the object exists.